variants/frontend-base/template.rb

```diff
@@ -149,6 +148,30 @@
 
 File.write("./package.json", JSON.generate(package_json))
 
+#fix js lint issues with generated defaults before linting runs
+gsub_file "babel.config.js",
+          "module.exports = function(api) {",
+          "module.exports = api => {",
+          force: true
+
+gsub_file "app/frontend/channels/index.js",
+          "/_channel\\.js$/",
+          "/_channel\\.js$/u"
+
+prepend_to_file "postcss.config.js" do
+  <<~ESLINTFIX
+  /* eslint-disable global-require */ 
+  'use strict'; \n
+  ESLINTFIX
+end
+
+prepend_to_file "postcss.config.js", "'use strict';"
```

Comment:
```suggestion

```

You're already adding `'use strict';` in the above `prepend_to_file`, so this is doubling up.